Core/DISET/private/BaseClient.py

```diff
@@ -291,6 +294,16 @@ def __checkThreadID( self ):
 
   def _connect( self ):
 
+    # Check if the useServerCertificate configuration changed
+    if gConfig.useServerCertificate() != self.useCertificates:
```

Comment:
No, the self.useCertificates is set in the constructor ( in self.__discoverCredentialsToUse() ) to True or False. So, it enters here only in case the flag changes after the client object instantiation.